arbitrum-docs/launch-orbit-chain/how-tos/customize-precompile.mdx

```diff
@@ -395,6 +396,166 @@ Here we can see that the gas cost incurred by the execution of the first transac
 
 To learn more about the gas cost model, see [how to estimate gas](/devs-how-tos/how-to-estimate-gas).
 
+## Option 5: Call and modify state.
+
+In this example, we will continue to introduce how to read and write your state in a precompile contract under ArbOS.
+
+First you need to open the [arbosstate.go][arbosstate_file_link] file, then find the [ArbosState][arbosstate_link] structure in this file. This is where the state stored in arbos is defined.
+
+We can define a state called `myNumber` here, `myNumber` uses the `storage.StorageBackedUint64` type. You can find more related types in [storage.go][storage_link]:
```

Comment:
```suggestion
Define a state key called `myNumber` of type `storage.StorageBackedUint64`. You can find more types in [storage.go][storage_link]:
```

_state_ or _state key_, or maybe _state variable_? Is this term formally standardized, or colloquially normalized?